src/main/webapp/dev/js/main/instructorFeedbackEdit.es6

```diff
@@ -505,7 +505,11 @@ function enableNewQuestion() {
 
     $newQuestionTable.find(`#rubricAddChoiceLink-${NEW_QUESTION}`).show();
     $newQuestionTable.find(`#rubricAddSubQuestionLink-${NEW_QUESTION}`).show();
-    $newQuestionTable.find(`#rubricWeights-${NEW_QUESTION}`).hide();
+    if (!hasAssignedWeights(NEW_QUESTION)) {
```

Comment:
I think the code will look more readable if this is inverted, i.e. -

``` js
if (hasAssignedWeights(NEW_QUESTION)) {
    $newQuestionTable.find(`#rubricWeights-${NEW_QUESTION}`).show();
} else {
    $newQuestionTable.find(`#rubricWeights-${NEW_QUESTION}`).hide();
}
```

Also add a new line before and after the `if-else` so that the code doesn't look cramped.